api/buckets/service.go

```diff
@@ -1685,15 +1702,11 @@ func (s *Service) unpinPath(ctx context.Context, path path.Path) error {
 	if err := s.IPFSClient.Pin().Rm(ctx, path); err != nil {
 		return err
 	}
-	node, err := s.IPFSClient.Unixfs().Get(ctx, path)
-	if err != nil {
-		return fmt.Errorf("getting node from path: %s", err)
-	}
-	size, err := node.Size()
+	stat, err := s.IPFSClient.Object().Stat(ctx, path)
 	if err != nil {
 		return fmt.Errorf("getting size of removed node: %s", err)
 	}
-	if err := s.sumBytesPinned(ctx, -size); err != nil {
+	if err := s.sumBytesPinned(ctx, int64(-stat.CumulativeSize)); err != nil {
```

Comment:
Realized cumulative size is the safest thing, since the size may not account for links.
This was applied in other places.